### PR TITLE
data NickAzureDevops

### PR DESCRIPTION
**Pull request by NickAzureDevops**

### CODE REVIEW

File: data/NickAzureDevops.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  "name": "Nicholas Chang",
+  "type": "personal",
+  "bio": "Platform Engineer, | Blogger, Azure and Community Advocate",
+  "tags": [
+    "Azure",
+    "Azure DevOps",
+    "Terraform",
+    "DevOps",
+    "Cloud Native",
+    "Github",
```
Comment (imbeshat) [inline data/NickAzureDevops.json:11]:
```suggestion
    "Github"
```

File: data/NickAzureDevops.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      "name": "Mastodon",
+      "url": "https://hachyderm.io/web/@nick_cloudops",
+      "icon": "FaCloud"
+    },
+     {
+      "group": "Socials",
+      "name": "Certifications",
+      "url": "https://www.credly.com/users/nicholas-chang.280d1eca/badges",
+      "icon": "FaCloud"
+    },
```
Comment (imbeshat) [inline data/NickAzureDevops.json:74]:
```suggestion
    }
```

**Review by imbeshat — CHANGES_REQUESTED** [review]

**Comment by NickAzureDevops** [comment]
Updated changes @imbeshat

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**